lib/msw-config.js

```diff
@@ -98,14 +98,14 @@ export default class MswConfig {
             queryParams[key] = value;
           });
           let request = Object.assign(
+              req,
```

Comment:
been a while since I actually looked at this. Req does not have a field named `requestBody` or `queryParams`. The fields in req are defined here https://mswjs.io/docs/api/request yet weirdly `body` isnt listed. What I believe is being done here is creating an object with the properties that mirage expects (it uses the property names that Pretender used since that is what it was written with). Given what Im seeing, switching the order should have no effect.  Does your `req` when it passes through here contain a property `requestBody` or `queryParams` that you always want to over write?

Reply:
It means that the msw MockedRequest `req` gets passed to the mirage handler, with the additonal `requestBody` and `queryParams` and not the POJO `{ requestBody, queryParams }` with the assigned enumerable properties of the MockedRequest `req`. So in the handler I can `request.body` and get the parsed multi part form data.

![request-in-mirage-handler](https://user-images.githubusercontent.com/12345/229193535-b96aa48e-8f09-4f8d-9c2a-e567eb8be5cd.png)

Reply:
Ahh, bear with me for one more question, shouldnt 
```
let request = Object.assign(
              {
                requestBody:
                    typeof req.body === 'string'
                        ? req.body
                        : JSON.stringify(req.body),
                queryParams: queryParams,
              },
              req
          );
```
 While not being req, but a POJO, still contain body because Object.assign should assign all properties in req to the POJO. Sounds like that is NOT happening which would explain the PR. 

Object.assign states `The Object.assign() method only copies enumerable and own properties from a source object to a target object.` so Im guessing body is either not enumerable or not an own property. 

If that is whats happening, I would agree that this PR does need to happen. The intent with the original code was to include properties like body into the passed object.

Reply:
Yeah, the body is a getter property and not enumerable and wasn't being assigned to the POJO